README.md

```diff
@@ -284,27 +288,24 @@ account.channelsPool.acquireChannel(channel => {
     });
 ```
 
-#### Transferring Kin to another account using whitelist service
-The Kin Blockchain also allows for transactions to be executed with no fee. Apps and services must first be approved, to learn more see [Going live with Kin](). If your service has been added to the whitelist you will be able to whitelist transactions for your clients.
-
-Clients will send an http request to your Node server containing their transaction, you can then whitelist it and return it to the client to send to the blockchain.
+#### Transferring Kin to Another Account Using Whitelist Service
+The Kin blockchain also allows for transactions to be performed with no fee. Apps and services have to be approved first (for details, see [Going live with Kin](). After your service has been added to the whitelist, you will be able to whitelist transactions for your clients. To have their transactions whitelisted, your clients will send HTTP requests containing their transactions to your Node server. You will then whitelist the transactions and return them to the clients to send to the blockchain. 
 
 ```javascript
 const whitelistedTransaction = account.whitelistTransaction(clientTransaction);
 ```
 
-Please note that if you are whitelisted, any payment sent from you (a server developed with the Node SDK) is already considered whitelisted, so there is no need for this step for the server transactions.
+Note that if you are whitelisted, any payment sent from a server developed with the Node SDK is already considered whitelisted, so the server transactions will not need the above step.
 
-### Decode_transaction
-When clients send you transactions for whitelisting they will be encoded. You can use `decodeTransaction` to read and then verify the contents.
+### Decoding Transactions
+When clients send you transactions for whitelisting, they will be encoded. It is recommended to decode each transaction and verify its details before whitelisting it. Use `decodeTransaction` to do that:
 
 ```javascript
 const decodedTransaction = client.decodeTransaction(encodedTransaction);
 ```
 
-#### Getting the minimum acceptable fee from the blockchain
-Transactions usually require a fee to be processed.
-To know what is the minimum fee that the blockchain will accept, use:
+#### Getting the Minimum Acceptable Fee from the Blockchain
+Transactions usually require a fee {to whom?} to be processed. The fee depends on how fast the transaction will be processed by the blockchain. To find out what the minimum acceptable fee is, use:
```

Comment:
@sbeskin to the blockchain, should also explain that the fee is specified in "Quarks" which is the smallest portion of kin i.e 0.00001 Kin.